am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -38,31 +37,37 @@ public static DefaultPermissionGrant createDefaultPermissionGrant(Set<Permission
             .serviceName(SERVICE_NAME)
             .resourceType(RESOURCE_TYPE)
             .resourceName(RESOURCE_NAME)
-            .attributePermissions(createPermissionsForAttribute(ROOT_ATTRIBUTE, permissions))
+            .attributePermissions(createPermissionsForAttribute(ROOT_ATTRIBUTE, permissions, PUBLIC))
+            .build();
+    }
+
+    public static DefaultPermissionGrant createDefaultPermissionGrant(JsonPointer attribute, Set<Permission> permissions) {
+        return DefaultPermissionGrant.builder()
```

Comment:
I'd prefer this overloaded method to delegate processing to main method for easier maintenance.

Reply:
done - I think this will be neatened further in my test refactor PRs. Coming to a screen near you this week.